Add workshop2023 (ignore branch name) (#2)
naims-aims/naims-aims.github.io: workshops/workshop2023.md

## About The Workshop

The Artificial Intelligence and Machine-Learning Subgroup (AIMS) of the North American Imaging in MS Cooperative (NAIMS) will be hosting a remote workshop focused on federated learning. Federated learning is a technique allowing for training of AI models across multiple servers/centers without directly sharing the underlying data. Rather than sharing the data, the model is shared between institutions instead, thereby circumventing the intrinsic challenges associated with data privacy and governance.

The purpose of the workshop is to gather a critical mass of people to brainstorm on suitable solutions for MS lesion segmentation in the context of a multi-site study. There will be a series of talks given by experts, with hands-on experience in federated learning for medical image segmentation, followed by a discussion session. 

Please mark it on your calendars! All levels of experience and backgrounds are welcome.

## Agenda

This will be a single day workshop. We hope that times will accomodate people on the west and east coast of North America.

**ALL TIMES EDT**:

| Time | Topic |
|---|---|
| 10:00 | Brief intro (what is FL, purpose of the workshop) |

> | 10:15 | Introduce Miro whiteboard: If ppl during the workshop have ideas, please enter them in the whiteboard (eg: ‘question’ category) |

Reviewer: ```suggestion
| 10:15 | Introduce Miro whiteboard: It will allow people having ideas/suggestions during the workshop to enter them in the whiteboard |
```

| 10:30 | Series of 4x20min talks (15min talk, 5min discussion) from people who have experience with medical FL and/or who have solutions |
| 12:00 | Break (zoom link stays open) |
| 13:00 | Series of 3x20min talks (15min talk, 5min discussion) from people who have experience with medical FL and/or who have solutions |
| 14:30 | Discussion, next steps, assign tasks |
| 17:00 | Adjourn |

## Speakers

> - Marco Lorenzi (INRIA): [https://fedbiomed.gitlabpages.inria.fr/](https://fedbiomed.gitlabpages.inria.fr/)

Reviewer: ```suggestion
- **Marco Lorenzi** (INRIA): [https://fedbiomed.gitlabpages.inria.fr/](https://fedbiomed.gitlabpages.inria.fr/)
```

> - Alexandros Karargyris (IHU Strasbourg): [https://www.medperf.org/](https://www.medperf.org/)

Reviewer: ```suggestion
- **Alexandros Karargyris** (IHU Strasbourg): [https://www.medperf.org/](https://www.medperf.org/)
```

> - Spyros Bakas (University of Pennsylvania): The Federated Tumor Segmentation initiative ([https://www.med.upenn.edu/cbica/fets/](https://www.med.upenn.edu/cbica/fets/))

Reviewer: ```suggestion
- **Spyros Bakas** (University of Pennsylvania): The Federated Tumor Segmentation initiative ([https://www.med.upenn.edu/cbica/fets/](https://www.med.upenn.edu/cbica/fets/))
```

> - Julien Cohen-Adad (Polytechnique Montreal / Mila): CODA ([https://github.com/coda-platform](https://github.com/coda-platform))

Reviewer: ```suggestion
- **Julien Cohen-Adad** (Polytechnique Montreal / Mila): CODA ([https://github.com/coda-platform](https://github.com/coda-platform))
```

> - Peter Chang (University of California, Irvine): TensorFlow framework for medical FL

Reviewer: ```suggestion
- **Peter Chang** (University of California, Irvine): TensorFlow framework for medical FL
```

> - Brennan Nichyporuk (McGill University / Mila): Generalization across cohorts/sources (and accounting for biases) for MS lesion ([https://arxiv.org/pdf/2108.00713.pdf](https://arxiv.org/pdf/2108.00713.pdf))

Reviewer: ```suggestion
- **Brennan Nichyporuk** (McGill University / Mila): Generalization across cohorts/sources (and accounting for biases) for MS lesion ([https://arxiv.org/pdf/2108.00713.pdf](https://arxiv.org/pdf/2108.00713.pdf))
```

> - Farhad Imam (Gates Ventures): GRIP Architecture

Reviewer: ```suggestion
- **Farhad Imam** (Gates Ventures): GRIP Architecture
```
